### PR TITLE
Feature/saved filters api

### PR DESCRIPTION
**Pull request by jdiouf**
added code to save filter searches

### CODE REVIEW

File: app/core/models.py [hunk 1]
```diff
@@ -352,3 +352,15 @@ class InterestList(TimeStampedModel):
                                          blank=True)
     name = models.CharField(max_length=200,
                             help_text="Enter the name of the list")
+
+
+class SavedFilter(TimeStampedModel):
+    """Model for Saved Filter"""
+
+    owner = models.ForeignKey(settings.AUTH_USER_MODEL,
```
Comment (mattdrayer) [inline app/core/models.py:360]: No pk / unique id?

**Review by mattdrayer — APPROVED** [review]
One question about primary keys for SavedFilter models, otherwise LGTM.

File: app/core/models.py (hunk 1 above)
Comment (jdiouf) [reply app/core/models.py:360]: All covered, Django creates an PK for us in this case.

**Review by KarenAJ — APPROVED** [review]
Approved!

**Comment by jdiouf** [comment]
@KarenAJ I didn't get a chance to add tests for the saved filters api. They should look very similar to the interest lists if you can add the rest when you get time. I wrote one test for creating saved filters; we need some for: editing owned, viewing owned, viewing all, deleting owned.

**Pull request merged**